dygraph.js

```diff
@@ -3602,19 +3602,25 @@ Dygraph.prototype.visibility = function() {
 };
 
 /**
- * Changes the visiblity of a series.
+ * Changes the visibility of one or more series.
  *
- * @param {number} num the series index
+ * @param {number|number[]} num the series index or an array of series indices
  * @param {boolean} value true or false, identifying the visibility.
  */
 Dygraph.prototype.setVisibility = function(num, value) {
   var x = this.visibility();
-  if (num < 0 || num >= x.length) {
-    console.warn("invalid series number in setVisibility: " + num);
-  } else {
-    x[num] = value;
-    this.predraw_();
+
+  if (num.constructor !== Array) num = [num];
```

Comment:
This LOC is blocking me to use this function. The constructor check, although fast, is not reliable. It can fall into certain corner cases which will fail to determine the variable as an array (false negative). E.g., when your array is from different window contexts. The foolproof solution is to use `Array.isArray()` method. See http://web.mit.edu/jwalden/www/isArray.html for more info.

As I'm using Dygraphs inside a GWT app, I'm falling into this corner case.